src/Posting/QR.coffee

```diff
@@ -352,6 +352,30 @@ QR =
     counter.hidden      = count < QR.max_comment/2
     (if count > QR.max_comment then $.addClass else $.rmClass) counter, 'warning'
 
+    splitPost = QR.nodes.splitPost
+    splitPost.hidden = count < QR.max_comment
+
+  splitPost: ->
+    count = QR.nodes.com.value.replace(/[\uD800-\uDBFF][\uDC00-\uDFFF]/g, '_').length
+    text = QR.nodes.com.value
+    return if count < QR.max_comment
```

Comment:
We should also probably bail out if `QR.selected.isLocked` so that users don't have unexpected consequences if they try to split the post after hitting submit.